### PR TITLE
Show Nethserver web interface URLs on login via /etc/issue

### PR DESCRIPTION
**Pull request by mrmarkuz**
After first reboot after base install the login URLs should be shown. This may be helpful when using DHCP or VMs.

Feature request:
https://community.nethserver.org/t/show-nethserver-web-interface-urls-on-login-via-issue/8560

### CODE REVIEW

File: src/lib/ks/interactive [hunk 1]
```diff
@@ -134,6 +134,9 @@ if [ -f /etc/nethserver-release ]; then
   echo "Fix /etc/issue..."
   cp -f /etc/nethserver-release /etc/issue
   echo -e 'Kernel \\r on an \\m\n' >> /etc/issue
+  echo 'To login to NethServer please use following URL(s):' >> /etc/issue
+  ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1' | sed 's/.*/https:\/\/&:980/' >> /etc/issue
```
Comment (DavidePrincipi) [inline src/lib/ks/interactive]: I'd use the `ip` command: it's output is more parsable than `ifconfig`.

File: src/lib/ks/interactive [hunk 2]
```diff
@@ -134,6 +134,9 @@ if [ -f /etc/nethserver-release ]; then
   echo "Fix /etc/issue..."
   cp -f /etc/nethserver-release /etc/issue
   echo -e 'Kernel \\r on an \\m\n' >> /etc/issue
+  echo 'To login to NethServer please use following URL(s):' >> /etc/issue
```
Comment (DavidePrincipi) [inline src/lib/ks/interactive]: A fixed `NethServer` string could not suit rebranded products (i.e. NethServer Enterprise). I'd rather write

    Access the web interface at the following URLs:

**Review by DavidePrincipi — CHANGES_REQUESTED** [review]
The `/etc/issue` file is overwritten by a template, defined in `nethserver-release` package. This "kickstart" version set up it temporarily: it's visible while the `system-init` event is running.

While `system-init` is running the web UI could be still not ready.

Is there a way to reset VT1 so that it shows the newly expanded `/etc/issue` ?

As alternative approach, you could also try to write directly to the system console. Indeed , if this information is required only at the end of the install procedure, it could be sent to the console/VT1 just once.

File: src/lib/ks/interactive (hunk 2 above)
Comment (mrmarkuz) [reply src/lib/ks/interactive]: It was intended to work only on first boot so no need for templating or writing directly to console as far as I understand.
I'll change the text to be neutral as you suggested.

File: src/lib/ks/interactive (hunk 1 above)
Comment (mrmarkuz) [reply src/lib/ks/interactive]: I have two alternatives for you, maybe they could be more optimized but they should work. What do you think?

ip addr | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | grep -v 127.0.0.1 | sed 's/.*/https:\/\/&:980/'

hostname -I | sed -E -e 's/[[:blank:]]+/\n/g' | sed 's/.*/https:\/\/&:980/' | grep -v 'https://:980'

Comment (DavidePrincipi) [reply src/lib/ks/interactive]: I prefer the `ip addr` one! With `-o` (oneline) option it's easier to parse:

    ip -o -4 addr | tr -s ' /' $'\t' | cut -f 4 | grep -vF 127.0.0.1 | xargs -I IP -- echo https://IP:980

**Review by DavidePrincipi — APPROVED** [review]

**Pull request merged**